people/gary-meyer.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: page
+title: Designer: Product, UI, UX, web.(hopefully a respectable dev at some point.)
+name: Gary Meyer
+github: @g4floyd
```

Comment:
Sounds good. I had no idea. I made the fix and commited the change. Should be good to go now.